Library/Formula/libmaxminddb.rb

```diff
@@ -44,9 +37,7 @@ def install
   end
 
   test do
-    system "curl", "-O", "http://geolite.maxmind.com/download/geoip/database/GeoLite2-Country.mmdb.gz"
-    system "gunzip", "GeoLite2-Country.mmdb.gz"
-    system "#{bin}/mmdblookup", "-f", "GeoLite2-Country.mmdb",
-                                "-i", "8.8.8.8"
+    system "#{bin}/mmdblookup", "-f", "#{buildpath}/t/maxmind-db/test-data/GeoIP2-City-Test.mmdb",
```

Comment:
You'll need to install this somewhere. The `buildpath` is only present during compile, the `testpath` is only present during the test. The two are sandboxed from each other.

Something like `(share/"examples").install buildpath/"t/maxmind-db/test-data/GeoIP2-City-Test.mmdb"` should be fine, at the end of `def install`.